neuralmonkey/config/parsing.py

```diff
@@ -143,84 +142,49 @@ def parse_value(string):
 
     return string
 
+def parse_config(config_file, filename=""):
```

Comment:
To že jsem to předtím pojmenoval nablble a že ty funkce nejlépe rozrvžené, ještě neznamená že se nemůžou jmenovat podle toho, co dělají. Ale jestli prohlašuješ, že s tím teď nehodláš nic dělat, tak bych to zmergoval.